concordance/concordance.cwl

```diff
@@ -0,0 +1,34 @@
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Concordance tool"
+baseCommand: ["python", "/opt/concordance/newConcordance.py"]
+inputs:
+    bam_1:
+        type: File
+        inputBinding:
+            position: 2
+        secondaryFiles: [.bai]
+    bam_2:
+        type: File
+        inputBinding:
+            position: 3
+        secondaryFiles: [.bai]
+    reference:
+        type: File
+        inputBinding:
+            position: 4
+        secondaryFiles: [.fai, "^.dict"]
+    snp:
+        type: File
+        inputBinding:
+            position: 5
+    output_file_name:
+        type: string?
+        inputBinding:
+            position: 6
```

Comment:
Based on the description of https://github.com/genome/concordance, it looks like this should have a prefix in its inputBinding for the parameter name:
```
prefix: "--output_geno"
```